packages/phrase/src/index.ts

```diff
@@ -1,2 +1,4 @@
 export { pull } from './pull-translations';
 export { push } from './push-translations';
+
+export const GLOBAL_KEY = 'global-key';
```

Comment:
We decided on putting this constant in index.ts yesterday.
However, I was thinking index.ts is more on re-exporting the named imports.
So for vocab pull I put it in config.ts, what is your thoughts?

Reply:
ah yep, i need to remove this as push-translations is now getting it from `./config`